Tests/LeafKitTests/GHTests/VaporLeaf.swift

```diff
@@ -94,21 +89,23 @@ final class GHLeafIssuesTestArchive: XCTestCase {
             try XCTAssertEqual(render(template, ["a": "a"]), expected)
         }
     }
-    
-    // TODO: This commenting style is not used anymore but needs a replacement
-    func _testGH127SingleLine() throws {
+
+    func testGH127SingleLine() throws {
```

Comment:
already tested in `LeafKitTests.testCommentSugar` and this test doesn't reflect the original issue anymore: https://github.com/vapor/leaf/issues/127

Reply:
@siemensikkema you're right, I've removed this